server/ast/drop_procedure.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2025 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ast
+
+import (
+	"fmt"
+
+	vitess "github.com/dolthub/vitess/go/vt/sqlparser"
+
+	"github.com/dolthub/doltgresql/postgres/parser/sem/tree"
+	pgnodes "github.com/dolthub/doltgresql/server/node"
+)
+
+// nodeDropProcedure handles *tree.DropProcedure nodes.
+func nodeDropProcedure(_ *Context, node *tree.DropProcedure) (vitess.Statement, error) {
+	if node == nil {
+		return nil, nil
+	}
+
+	if node.DropBehavior == tree.DropCascade {
+		return nil, fmt.Errorf("DROP PROCEDURE with CASCADE is not supported yet")
+	}
+
+	if len(node.Procedures) == 0 {
+		return nil, fmt.Errorf("no function name specified for DROP PROCEDURE")
+	}
+
+	return vitess.InjectedStatement{
+		Statement: pgnodes.NewDropProcedure(
+			node.IfExists,
+			node.Procedures,
+			node.DropBehavior == tree.DropCascade),
```

Comment:
Always false according to the above

Reply:
That is correct. This is just here so that, when we _eventually_ add support, this will already be set correctly.